packages/site/src/ui/page-heading.js

```diff
@@ -53,3 +31,29 @@ PageHeading.propTypes = {
   children: PropTypes.node,
   packageName: PropTypes.string
 }
+
+const BadgeSpacer = () => <span style={{ width: layout.spacingLarge }} />
+
+function Beta(props) {
+  return (
+    <>
+      <BadgeSpacer />
+      <Badge color={Badge.colors.blue}>Beta</Badge>
```

Comment:
@sweeting we missed one

Reply:
No comprendo.  Missed one what?

Reply:
color change. i'm guess it should be colorsBlue.base or something

Reply:
Still `blue`, see: https://design-system.pluralsight.com/components/badge#colors-&-appearance